.github/actions/e2e-test/generate-test-data.sh

```
@@ -1,5 +1,7 @@
 #!/bin/bash
 
+DB_SETUP_SQL_PATH=${DB_SETUP_SQL_PATH:-e2e-tests/cypress/db-setup/mattermost.sql}
+
 psql -d $TEST_DATABASE_URL -v "ON_ERROR_STOP=1" -c "CREATE DATABASE migrated;";
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)